assets/src/__tests__/components/__snapshots__/Label.test.js.snap

```diff
@@ -6,13 +6,18 @@ exports[`renders correctly 1`] = `
     Object {
       "color": "blue",
       "display": "inline-block",
-      "left": 5,
+      "left": "undefined%",
```

Comment:
What is the reason for this `undefined` I am not sure if that is true in the real world?

Reply:
Hmm, i guess in some test case there is no value being passed in.

Reply:
I think we need to understand if that is true and what is that test case? We might need to change the test case if that is not true

Reply:
Ah I didn't understand how the tests were setup, I believe I need to change how that one test is configured.

Reply:
I don't think we are testing all possible test cases with Label. I am not insisting on writing more tests. I feel understanding what use case is being tested is important.

Reply:
@justin0022 Can you give your thoughts on this? we are planning for a release next week and your input might help us on this

Reply:
I updated the test definition

Reply:
I guess @justin0022 @chrisrrowland solved the issue.